backend/user/management/commands/send_shadow_notifs.py

```diff
@@ -28,24 +28,27 @@ def add_arguments(self, parser):
         parser.add_argument("message", type=str, help="JSON-formatted message to send")
 
         # optional argument
-        parser.add_argument("--usernames", type=str, help="list of usernames")
+        parser.add_argument("--users", type=str, help="list of usernames")
```

Comment:
nit: Suggest writing list of pennkeys just to be explicit to non-django developers